tests/cli/test_integration_datasets.py

```diff
@@ -396,15 +396,21 @@ def test_dataset_import_renkulab_dataset_with_image(runner, project, client, cli
 @pytest.mark.integration
 @retry_failed
 @pytest.mark.vcr
-def test_import_renku_dataset_preserves_directory_hierarchy(runner, project, client, load_dataset_with_injection):
+@pytest.mark.parametrize(
+    "datadir_option,expected_datadir", [([], Path(DATA_DIR) / "remote"), (["--datadir", "mydir"], Path("mydir"))]
```

Comment:
How does `DATA_DIR` get its value here? Is it a fixture?

Reply:
It's imported at the top `from renku.core.management.repository import DEFAULT_DATA_DIR as DATA_DIR`

Reply:
🤦 my bad I completely missed that. I thought I fully expanded the diffs but I guess I did not.